components/org.wso2.carbon.identity.api.user.association/org.wso2.carbon.identity.api.user.association.v1/src/main/java/org/wso2/carbon/identity/rest/api/user/association/v1/impl/UserIdApiServiceImpl.java

```diff
@@ -0,0 +1,53 @@
+package org.wso2.carbon.identity.rest.api.user.association.v1.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.wso2.carbon.identity.rest.api.user.association.v1.UserIdApiService;
+import org.wso2.carbon.identity.rest.api.user.association.v1.core.UserAssociationService;
+import org.wso2.carbon.identity.rest.api.user.association.v1.dto.AssociationRequestDTO;
+import org.wso2.carbon.identity.rest.api.user.association.v1.dto.UserDTO;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.List;
+import javax.ws.rs.core.Response;
+
+import static org.wso2.carbon.identity.rest.api.user.association.v1.AssociationEndpointConstants.ASSOCIATIONS_PATH;
+
+public class UserIdApiServiceImpl extends UserIdApiService {
+
+    @Autowired
+    private UserAssociationService userAssociationService;
+
+    @Override
+    public Response userIdAssociationsDelete(String userId) {
+
+        userAssociationService.deleteUserAccountAssociation(userId);
```

Comment:
userId= base64encode(Userstore/username) format. Hence, decode

Reply:
will do this later.